packages/metro/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './bundle';
+export * from './config';
+export * from './start';
```

Comment:
I have mixed feelings about this approach, since it makes it hard to actually track down what this module exports at the end of the day.

Since we're always mindful about what kind of API surface we explore, how about we do it explicitly, like in other modules: https://github.com/react-native-community/cli/blob/7d460229962fff69841b6e743c73fd0b1d5dc529/packages/platform-ios/src/index.ts#L5-L7

Reply:
I made it explicit.

Reply:
Thanks. Are you working on the rest of the things I left as a PR review? Or is this the final round to review once again?

Reply:
And I limited the exports to only what I need to consume from this PR, as well as what the cli project needs to consume (the `command` objects).